### PR TITLE
[TRTLLM-13613][test] Trim duplicated and dead multimodal accuracy tests from pre-merge CI

### PR DESCRIPTION
**Pull request by Wanli-Jiang**
# Multimodal Accuracy Tests — Change Summary

Trim duplicated / dead / over-budget multimodal accuracy tests from **pre-merge** CI.
Runtimes = OpenSearch L0 CI, 45-day avg. **No input modality (image/video/audio)
coverage is lost** — every change keeps a cheaper or equivalent guard (logit-match
unittest, EPD test, post-merge, or QA).

Reflects committed state as of `b4420a5908`.

| ID | Test (Class::variant) | Change | Stage before → after | Runtime/PR | Why |
|----|------------------------|--------|----------------------|-----------:|-----|
| A1 | `TestNanoV3Omni::test_auto_dtype` VideoMME | drop VideoMME from **nvfp4** only; **keep on fp8** | fp8 unchanged (MMMU+VoxPopuli+VideoMME, pre-merge H100); nvfp4 task spec → MMMU+VoxPopuli | folded into A3 | Reviewer (@2ez4bz): keep VideoMME on one agg flavor — it's the only video coverage on the standard (non-EPD) path. fp8 chosen (more stable in CI than nvfp4); nvfp4 VideoMME still covered by EPD. |
| A2 | `test_llm_api_pytorch_multimodal.py::TestKimiK25::test_nvfp4[dep8]` | demote | pre-merge → post-merge, 8× B200 (`l0_dgx_b200.yml`) | ~1934 s | most expensive mm test (~32 min) every PR; low per-PR value. Coverage retained post-merge + QA (`llm_function_core.txt:819` — confirmed re @xinhe-nv: already QA-listed, no new QA entry needed). |
| A3 | `TestNanoV3Omni::test_auto_dtype[nvfp4]` | demote | pre-merge → post-merge, 1× B200 (`l0_b200.yml`) | ~379 s | per-PR nvfp4 image/audio regression value is low; `[fp8_mmmu_encoder_cuda_graph]` keeps a pre-merge B200 image guard, `[bf16]` is already post-merge, and fp8 stays pre-merge (H100). Coverage retained post-merge + QA. |
| B1 | `TestNemotron_Nano_12B_V2_VL::test_auto_dtype[full_budget]` | delete param | listed nowhere → removed | 0 s | dead variant; `forced_chunked_prefill` sibling covers the model |
| B2 | `TestQwen3VL::test_auto_dtype[full_budget]` | delete param | listed nowhere → removed | 0 s | dead variant; sibling covers the model |
| B3 | `TestMistralSmall24B::test_auto_dtype[full_budget]` | delete param (+ unused `MAX_NUM_TOKENS`) | listed nowhere → removed | 0 s | dead variant; sibling covers the model |
| B4 | `TestKimiK25::test_nvfp4[tp8] / [tp8_attn_dp] / [ep8]` | delete params (mm file) | listed nowhere → removed | 0 s | dead in mm file; text-path namesake in `test_llm_api_pytorch.py` runs these. QA only ever referenced mm `[dep8]` (`llm_function_core.txt:819`), so no dangling QA entries. |
| C1 | `TestQwen2_5_VL_7B::test_auto_dtype` | remove from pre-merge | pre-merge (L40S) → QA only | 689 s | "MMMU sanity" overlaps cheaper logit-match unittest (109 s) on same stage; QA retains MMMU |
| C2 | `TestVILA1_5_3B::test_auto_dtype` | remove from pre-merge | pre-merge (L40S) → QA only | 291 s | same; logit-match `modeling_vila` (38 s) stays pre-merge; QA retains MMMU |
| C3 | `examples/test_multimodal.py::...[video-neva]` + `[kosmos-2]` | remove from pre-merge | pre-merge (L40S) → removed | 0 s | legacy **TRT** workflow; 0 runs in 120 days; PyTorch serving e2e tests retained |

**Files touched**
- `tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py` — A1, B1–B4
- `tests/integration/test_lists/test-db/l0_b200.yml` — A3
- `tests/integration/test_lists/test-db/l0_dgx_b200.yml` — A2
- `tests/integration/test_lists/test-db/l0_l40s.yml` — C1–C3

**Per-PR pre-merge time recovered:** ~980 s (L40S) + ~1934 s (8× DGX_B200, A2) + ~379 s
(1× DGX_B200, A3). VideoMME stays on fp8/H100, so there is no DGX_H100 recovery.

## Summary by CodeRabbit

* **Tests**
  * Updated multimodal and accuracy test coverage to focus on the currently supported evaluation scenarios.
  * Simplified several benchmark runs to use a single, standard configuration, reducing redundant test variants.
  * Adjusted test scheduling so multimodal checks run in the appropriate pipeline stage.

## PR Checklist

Please review the following before submitting your PR:
- PR description clearly explains what and why. If using CodeRabbit's summary, please make sure it makes sense.
- PR Follows [TRT-LLM CODING GUIDELINES](https://github.com/NVIDIA/TensorRT-LLM/blob/main/CODING_GUIDELINES.md) to the best of your knowledge.
- Test cases are provided for new code paths (see [test instructions](https://github.com/NVIDIA/TensorRT-LLM/tree/main/tests#1-how-does-the-ci-work))
- If PR introduces API changes, an appropriate PR label is added - either `api-compatible` or `api-breaking`. For `api-breaking`, include `BREAKING` in the PR title.
- Any new dependencies have been scanned for license and vulnerabilities
- [CODEOWNERS](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/CODEOWNERS) updated if ownership changes
- Documentation updated as needed
- Update [tava architecture diagram](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/tava_architecture_diagram.md) if there is a significant design change in PR.
- The reviewers assigned automatically/manually are appropriate for the PR.

- [x] Please check this after reviewing the above items as appropriate for this PR.

## GitHub Bot Help

To see a list of available CI bot commands, please comment `/bot help`.

### CODE REVIEW

File: tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     skip_pre_blackwell,
     skip_pre_hopper,
 )
-from .accuracy_core import MMMU, LlmapiAccuracyTestHarness, VideoMME, VoxPopuli
```
Comment (2ez4bz) [inline tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py:37]: Can we keep `VideoMME`? We have zero video coverage otherwise.

**Review by 2ez4bz — APPROVED** [review]

File: tests/integration/test_lists/test-db/l0_dgx_b200.yml [hunk 2]
```diff
@@ -219,6 +218,7 @@ l0_dgx_b200:
   - accuracy/test_llm_api_pytorch.py::TestKimiK25::test_nvfp4[tp8_attn_dp] TIMEOUT (60)
   - accuracy/test_llm_api_pytorch.py::TestKimiK25::test_nvfp4[ep8] TIMEOUT (60)
   - accuracy/test_llm_api_pytorch.py::TestKimiK25::test_nvfp4[dep8] TIMEOUT (60)
+  - accuracy/test_llm_api_pytorch_multimodal.py::TestKimiK25::test_nvfp4[dep8] TIMEOUT (120)
```
Comment (xinhe-nv) [inline tests/integration/test_lists/test-db/l0_dgx_b200.yml:212]: do we need to add this case into QA's test list?

**Review by xinhe-nv — APPROVED** [review]

File: tests/integration/test_lists/test-db/l0_dgx_b200.yml (hunk 2 above)
Comment (Wanli-Jiang) [reply tests/integration/test_lists/test-db/l0_dgx_b200.yml:212]: we already have that in QA test list.

File: tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py (hunk 1 above)
Comment (Wanli-Jiang) [reply tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py:37]: keep fp8 and move nvfp4 tests to post-merge as discussed.

**Review by moraxu — APPROVED** [review]
